crates/nu-command/src/filesystem/save.rs

```diff
@@ -499,7 +500,10 @@ fn stream_to_file(
                     let len = buf.len();
                     reader.consume(len);
                     bytes_processed += len as u64;
-                    bar.update_bar(bytes_processed);
+                    if last_update.elapsed() >= Duration::from_millis(75) {
```

Comment:
Just curious, why 75?

Reply:
Based purely on what looked good to me. I thought 50ms was too fast and 100ms was a bit too slow.